composer.json

```
@@ -35,7 +35,7 @@
         "psr/log": "^1.1 || ^2.0 || ^3.0",
         "sonata-project/block-bundle": "^4.11",
         "sonata-project/doctrine-extensions": "^1.8 || ^2.0",
-        "sonata-project/exporter": "^2.11 || ^3.0",
+        "sonata-project/exporter": "^3.1",
```

```suggestion
        "sonata-project/exporter": "^2.14 || ^3.1.1",
```

We need to keep the v2 support for the PHP 7.4 support.
I added the Interface on the v2 of exporter because it was simpler.

---

agree, v2 is added